.gitignore

```diff
@@ -74,15 +74,15 @@ dynawo/3rdParty/adept/adept-1.1.tar.gz
 dynawo/3rdParty/suitesparse/SuiteSparse-4.5.4.tar.gz
 dynawo/3rdParty/nicslu/_nicslu301.zip
 dynawo/3rdParty/sundials/sundials-2.7.0.tar.gz
-util/curvesToHtml/ressources/arrow-down.gif
-util/curvesToHtml/ressources/arrow-left.gif
-util/curvesToHtml/ressources/arrow-right.gif
-util/curvesToHtml/ressources/arrow-up.gif
-util/curvesToHtml/ressources/jquery.flot.crosshair.js
-util/curvesToHtml/ressources/jquery.flot.js
-util/curvesToHtml/ressources/jquery.flot.navigate.js
-util/curvesToHtml/ressources/jquery.flot.selection.js
-util/curvesToHtml/ressources/jquery.js
+util/curvesToHtml/resources/arrow-down.gif
```

Comment:
Did you rename the folder? (if so it seems to be hidden in the review so I prefer to double-check)

Reply:
No I ill do it